src/test/scala/io/iohk/ethereum/ledger/LedgerTestingUtils.scala

```diff
@@ -0,0 +1,9 @@
+package io.iohk.ethereum.ledger
+
+object LedgerTestingUtils {
+  def calculateFullMinerReward(blockRewardCalculator: BlockRewardCalculator, blockNumber: BigInt, numberOfOmmers: Int): BigInt = {
```

Comment:
why not something like:
```
package io.iohk.ethereum.ledger

object BlockRewardCalculatorOps {

  implicit class BlockRewardCalculatorOps(calculator: BlockRewardCalculator) {

    def calculateFullMinerReward(blockNumber: BigInt, numberOfOmmers: Int): BigInt = {
      val rewardForBlock = calculator.calculateMiningRewardForBlock(blockNumber)
      val rewardForOmmers = calculator.calculateMiningRewardForOmmers(blockNumber, numberOfOmmers)
      rewardForBlock + rewardForOmmers
    }
  }
}
```
Then, in tests:
```
import io.iohk.ethereum.ledger.BlockRewardCalculatorOps._
```

disclaimer: Maybe some adjustments are needed in order to compile.

Reply:
Interesting fact, same name for implicit class and object wrapping it doesn't play well together